models/classes/user/LtiUserService.php

```diff
@@ -72,6 +73,8 @@ public function findOrSpawnUser(LtiLaunchData $launchData)
             $taoUser = $this->findUser($launchData);
             if ($taoUser === null) {
                 $taoUser = $this->spawnUser($launchData);
+
+                $this->getLtiUserEventDispatcher()->dispatch($taoUser);
```

Comment:
Shouldn't event be dispatched at `\oat\taoLti\models\classes\user\LtiUserFactoryService::create` ?

Reply:
I do not think a factory should dispatch events. This place were I put dispatch the event regardless the LtiUser type, but only consider Learner